Source/WebKit/UIProcess/API/glib/WebKitInstallMissingMediaPluginsPermissionRequest.cpp

```diff
@@ -152,18 +111,14 @@ WebKitInstallMissingMediaPluginsPermissionRequest* webkitInstallMissingMediaPlug
  * Returns: a string with the description provided by the media backend.
```

Comment:
Another example of how I like to do doc comments for lobotomized APIs:

```
/**
 * webkit_install_missing_media_plugins_permission_request_get_description:
 * @request: a #WebKitInstallMissingMediaPluginsPermissionRequest
 *
 * This function returns an empty string.
 *
 * Returns: an empty string
 *
 * Since: 2.10
 *
 * Deprecated: 2.40
 */
 ```